frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature

```diff
@@ -273,3 +273,21 @@ Feature: Topology chart area
               And user sees shortcut for Access create connector handle
               And user sees shortcut for Qpen quick search modal
               And user sees shortcut for Drag and drop a JAR file into Topology
+
+
+        @regression @manual @5722
+        Scenario: Display of External Bindable resources: T-06-TC24
+            Given user has created external bindable resource
+             When user goes to Topology chart view
+             Then user will see the bindable resource in trapezoid shape
```

Comment:
This scenario will also require us to install SBO

Reply:
Added the step for it